package.json

```diff
@@ -25,10 +25,11 @@
     "danger": "danger-ci",
     "lint": "eslint '@(packages|scripts)/**/{*.js,package.json}' --ignore-pattern node_modules --ignore-pattern storybook-dist",
     "postbuild": "rimraf \"./packages/*/dist/{,**/}__*__\"",
-    "prettier": "prettier --write '@(packages|scripts)/**/*.@(js|css)' '*.js'",
+    "prepare": "node scripts/monorepo-introduction.js",
+    "prettier": "prettier --loglevel warn --write '@(packages|scripts)/**/*.@(js|css)' '*.js'",
     "prettier:validate": "prettier-check '@(packages|scripts)/**/*.@(js|css)' '*.js'",
     "prettier:check": "prettier --list-different '@(packages|scripts)/**/*.@(js|css)' '*.js'",
-    "stage:venia": "yarn workspace @magento/venia-concept run start; cd - >/dev/null",
+    "stage:venia": "yarn workspace @magento/venia-concept run start",
```

Comment:
This breaks for me now. Upon loading root after a clearing cache:

[![Image from Gyazo](https://i.gyazo.com/b670ea1257392d23ced039cf95f09b81.png)](https://gyazo.com/b670ea1257392d23ced039cf95f09b81)